airbyte-integrations/connectors/source-instagram/source_instagram/streams.py

```diff
@@ -358,8 +358,9 @@ def _get_children(self, ids: List):
 class MediaInsights(Media):
     """Docs: https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights"""
 
-    MEDIA_METRICS = ["engagement", "impressions", "reach", "saved"]
-    CAROUSEL_ALBUM_METRICS = ["carousel_album_engagement", "carousel_album_impressions", "carousel_album_reach", "carousel_album_saved"]
+    MEDIA_METRICS = ["total_interactions", "impressions", "reach", "saved", "video_views", "likes", "comments", "shares"]
```

Comment:
@artem1205 I see `total_interactions` is listed as an alternative, but the [change log ](https://developers.facebook.com/docs/instagram-api/changelog)states this:

> Note: total_interactions, which is listed as an alternative for some of the deprecated metrics, is currently only available using version 18.0 and does not work with older versions. When querying older versions before Dec 11, 2023, please use the engagement metric

This seems to imply total_interactions won't work for us since I believe we're on v11 - is this right?

Reply:
According to the docs, this should not work with v11, but it does (tested with customers creds).